build.sh

```diff
@@ -39,13 +43,13 @@ if [ "${chart_dir}" != "" ]; then
 fi
 
 echo "Getting the desired marketplace Docker image..."
-MARKETPLACE_TOOLS_TAG="0.7.0"
+MARKETPLACE_TOOLS_TAG="0.10.0"
 LOCAL_MARKETPLACE_TOOLS_TAG="local-$USER"
 docker pull "gcr.io/cloud-marketplace-tools/k8s/dev:$MARKETPLACE_TOOLS_TAG"
 docker tag "gcr.io/cloud-marketplace-tools/k8s/dev:$MARKETPLACE_TOOLS_TAG" \
            "gcr.io/cloud-marketplace-tools/k8s/dev:$LOCAL_MARKETPLACE_TOOLS_TAG"
 
 echo "Building/verifying app..."
-make -j4 -e app/verify
+make -j4 -e $build_target
```

Comment:
I think the var should be quoted here since it can expand into multiple args otherwise